### ISSUE TITLE
CMSUno v1.5.2 has a XSS vulnerability in title.

### ISSUE THREAD

**Issue by SunJ3t** [boiteasite/cmsuno#7]
```
A xss vulnerability was discovered in CMSUno v1.5.2.
There is a persistent XSS vulnerability in title[titre] if I use the payload <script>alert(1)</script>.
```

![cmsuno1](https://user-images.githubusercontent.com/27290132/43942006-e2b696dc-9ca8-11e8-992e-d217280cc15a.png)

![cmsuno2](https://user-images.githubusercontent.com/27290132/43942032-fed2be40-9ca8-11e8-9c95-d116b5577ea8.png)

**Comment by boiteasite** [comment]
Hi,

Fixed with new update 1.5.3.
Thanks for this report.

Regards

**Comment by fgeek** [comment]
CVE-2018-15567 has been assigned for this issue.
